mpm/main.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path 
+from dotenv import load_dotenv, find_dotenv
+
+import os
+import crawler
+
+# Get env args
+def get_args():
+
+    try:
+        env_path = Path('.') / '.env'
+        load_dotenv(dotenv_path=env_path)
+    except IOError:
+        print("No .env file to load")
+
+    try:
+        month = os.getenv("MONTH")
+    except BaseException as e:
+        print("Invalid month: {}" .format(e))
+        raise e 
+
+    try:
+        year = os.getenv("YEAR")
+    except BaseException as e:
+        print("Invalid month: {}" .format(e))
+        raise e 
```

Comment:
Quando isso for para o main, aqui vale a pena lançar o os.Exit ou algo assim que você consiga, além de escrever o erro, usar um código de erro específico

Isso pois os estágios no dadosjus obedecem a uma API de retorno, tipo os códigos de retorno do protocolo HTTP. 

Aqui tem um resumo: https://github.com/dadosjusbr/coletores/tree/master/status

No caso   dos parâmetros inválidos, o  status code de erro é 1.